quiz.js

```diff
@@ -0,0 +1,64 @@
+// Create object to hold user inputs
+var inputObject = {
+  height: "",
+  character: ""
+};
+
+// Reference to Object elements
+var height = inputObject.height;
+var characer = inputObject.character;
+
+// Reference to elements on page
+var formElement = document.getElementById("form"); //to listen for field inputs
+var heightElement = document.getElementById("height");
+var characterElement = document.getElementById("character");
+var buttonElement = document.getElementById("button");
+
+// Error Message
+var alertMissingInput = "Sorry, please fill out both fields first.";
+
+// Attached event listener to button. If you click it, call PushInputs function
+buttonElement.addEventListener("click", pushInputs);
+
+// Submit form if you press enter while in a field. 
+formElement.addEventListener("keyup", function(event) {
+    if (event.keyCode == 13) { // 13 is key for enter key
+        pushInputs();
+    }
+});
+
+// Function to push user inputs into object
+function pushInputs() {
+  // Save new inputs
+  height = heightElement.value;
+  character = characterElement.value;
+  // Error if either of new inputs are nothing
+  if (character == "" || height == "") {
+    alert(alertMissingInput);
+  } else {
+    makeTree(height, character);
```

Comment:
Instead of passing individual variables into this function, you should be passing the entire object.